.github/workflows/ci-test-helm-charts.yml

```diff
@@ -15,7 +15,7 @@ on:
 jobs:
   lint:
     name: Helm Chart Tests / ubuntu 20.04
```

Comment:
Please change the job name from `Helm Chart Tests / ubuntu 20.04` to `Helm Chart Tests`.